Add Fall 2019 Mentorship newspost (#2503)
ppy/osu-wiki: news/2019-09-07-community-mentorship-program-fall-2020-mentee-signups-now-open.md

---
layout: post
title: "Community Mentorship Program Fall 2020 Signups Now Open"
date: 2019-09-08 12:50:00 +0000
---

The next cycle of the Community Mentorship program is starting again! If you've ever found yourself wanting to get into beatmapping, or expanding your current knowledge and skills, this program will serve as a great opportunity for you!

![](/wiki/shared/news/banners/community-mentorship-program.jpg)

The Community Mentorship program serves as place for upcoming mappers and modders to develop their skills underneath the guidance of more experienced mappers and modders in the community. Over the course of a cycle, you'll have the opportunity to learn from your mentor, participate in various mentorship events, such as mapping contests, hang out with fellow mentees and more!

> For those of you already experienced in mapping, and have a desire to take those that are newer under your wing, mentor applications are already open! In the forum announcement and rule thread [here] (https://osu.ppy.sh/community/forums/topics/955356), you'll be able to apply if you’re considering mentoring! Keep in mind mentor applications take sometime to process, so please be patient.

Reviewer: ```suggestion
For those of you already experienced in mapping, and have a desire to take those that are newer under your wing, mentor applications are already open! In the forum announcement and rule thread [here](https://osu.ppy.sh/community/forums/topics/955356), you'll be able to apply if you’re considering mentoring! Keep in mind mentor applications take sometime to process, so please be patient.
```

Remove the space from...
```
[here] (https
      ^ here
```

For this upcoming fall cycle, we are looking for both mentors and mentees for all game modes!

> Since last cycle, we've restructured our general mentorship timeline.

Author: yup looks cleaner than previous ones

- No longer are mentors required to sign-up each cycle, instead a mentor may mentor for as long as they'd like, but can retire at any time, once completing their commitments to their mentees.

- Mentee applications are accepted every 4-weeks, and expire after 8, if they have not been accepted. However, applicants with high interests are encouraged to reapply as often as possible, to better reflect their interest, changes in mentor preferences, and availability.

- Mentors and mentees are fixed for a minimum 8 weeks, but mentors can also hold mentees for longer than this, and are encouraged to do so.

> For this following cycle, the timeline is as follows (all opening and closing dates start and end at midnight UTC+0):

Reviewer: I think adding one before this paragraph would be nice as well.
```
- Mentors and mentees are fixed for a minimum 8 weeks, but mentors can also hold mentees for longer than this, and are encouraged to do so.

---

For this following cycle, the timeline is as follows (all opening and closing dates start and end at midnight UTC+0):
```

Author: sure

> Mentor sign-ups have started  2019-09-01 and close on 2019-09-08.

Reviewer: ```suggestion
- Mentor sign-ups have started  2019-09-01 and close on 2019-09-08.
```

Should probably use a bulleted list instead. (for the next two as well)

> Mentee sign-ups will open on 2019-09-22 and close on 2019-10-03.

Reviewer: ```suggestion
- Mentee sign-ups will open on 2019-09-22 and close on 2019-10-03.
```

> The fall 2019 cycle will start on 2019-10-06 to 2020-01-05.

Reviewer: ```suggestion
- The fall 2019 cycle will start on 2019-10-06 to 2020-01-05.
```

Throughout this active cycle time, the organization team will host various events on the Mentorship discord server, engaging and offering events for mentees and mentors to participate in. These include feedback lobbies, mapping contests, and more! Some of our previous puzzles can be found here at this [thread](https://osu.ppy.sh/community/forums/topics/565382), and previous discussions can be found [here](https://osu.ppy.sh/community/forums/topics/514122). 

> If any of this interests you or anyone you know, please go to our [forum announcement and rules thread(https://osu.ppy.sh/community/forums/topics/955356)! 

Reviewer: ```suggestion
If any of this interests you or anyone you know, please go to our [forum announcement and rules thread](https://osu.ppy.sh/community/forums/topics/955356)! 
```
missing `]` for link

---
layout: post

> title: "Community Mentorship Program Fall 2020 Signups Now Open"

Reviewer 2: fall 2019, rename file too

Author: ye

date: 2019-09-08 12:50:00 +0000
---

> The next cycle of the Community Mentorship program is starting again! If you've ever found yourself wanting to get into beatmapping, or expanding your current knowledge and skills, this program will serve as a great opportunity for you!

Reviewer 2: ```suggestion
The next cycle of the Community Mentorship program is starting again! If you've ever found yourself wanting to get into beatmapping or expand your current knowledge and skills, this program will serve as a great opportunity for you!
```

Author: current wording works better here, sounds more natural

Reviewer 2: you're trying to say that the reader wants to expand their knowledge, right? in that case u want normal present tense instead of continuous

you need these two bracketed parts to match tense:
"wanting to [get into beatmapping] or [expand your current knowledge and...]"

Author: okay sure why not

![](/wiki/shared/news/banners/community-mentorship-program.jpg)

> The Community Mentorship program serves as place for upcoming mappers and modders to develop their skills underneath the guidance of more experienced mappers and modders in the community. Over the course of a cycle, you'll have the opportunity to learn from your mentor, participate in various mentorship events, such as mapping contests, hang out with fellow mentees and more!

Reviewer 2: ```suggestion
The Community Mentorship program serves as place for upcoming mappers and modders to develop their skills underneath the guidance of more experienced mappers and modders in the community. Over the course of a cycle, you'll have the opportunity to learn from your mentor, participate in various mentorship events (such as mapping contests), hang out with fellow mentees, and more!
```

> For those of you already experienced in mapping, and have a desire to take those that are newer under your wing, mentor applications are already open! In the forum announcement and rule thread [here](https://osu.ppy.sh/community/forums/topics/955356), you'll be able to apply if you’re considering mentoring! Keep in mind mentor applications take sometime to process, so please be patient.

Reviewer 2: ```suggestion
For those of you who are already experienced in mapping and have a desire to take those that are newer under your wing, mentor applications are open! In the [forum announcement and rule thread](https://osu.ppy.sh/community/forums/topics/955356), you can find more details about how to apply. Please keep in mind that mentor applications take some time to process, so you'll need to be patient.
```

Reviewer 2: well according to this post they're already closed, so you'll need to rewrite or remove this. not sure why the news post is being made so late

Author: mhm yeah edited to reflect closed mentor apps

---

For this upcoming fall cycle, we are looking for both mentors and mentees for all game modes!

> Since last cycle, we've restructured our general mentorship timeline.

Reviewer 2: ```suggestion
Since last cycle, we've restructured our general mentorship timeline:
```

> - No longer are mentors required to sign-up each cycle, instead a mentor may mentor for as long as they'd like, but can retire at any time, once completing their commitments to their mentees.

Reviewer 2: ```suggestion
- Mentors are no longer required to sign up each cycle; instead, a mentor may stay for as long as they'd like, and can retire at any time once completing their commitments to their mentees.
```

> - Mentee applications are accepted every 4-weeks, and expire after 8, if they have not been accepted. However, applicants with high interests are encouraged to reapply as often as possible, to better reflect their interest, changes in mentor preferences, and availability.

Reviewer 2: ```suggestion
- Mentee applications are processed every 4 weeks, and expire after 8 if they have not been accepted. However, applicants with high interest in the program are encouraged to reapply as often as possible to better reflect their interest, changes in mentor preferences, and availability.
```

> - Mentors and mentees are fixed for a minimum 8 weeks, but mentors can also hold mentees for longer than this, and are encouraged to do so.

Reviewer 2: ```suggestion
- Mentors and mentees are fixed for a minimum of 8 weeks, but mentors can hold mentees for longer than this and are encouraged to do so.
```

---

For this following cycle, the timeline is as follows (all opening and closing dates start and end at midnight UTC+0):

> - Mentor sign-ups have started  2019-09-01 and close on 2019-09-08.

Reviewer 2: ```suggestion
- Mentor sign-ups opened on 2019-09-01 and closed on 2019-09-08.
```

- Mentee sign-ups will open on 2019-09-22 and close on 2019-10-03.

> - The fall 2019 cycle will start on 2019-10-06 to 2020-01-05.

Reviewer 2: ```suggestion
- The fall 2019 cycle will run from 2019-10-06 to 2020-01-05.
```

> Throughout this active cycle time, the organization team will host various events on the Mentorship discord server, engaging and offering events for mentees and mentors to participate in. These include feedback lobbies, mapping contests, and more! Some of our previous puzzles can be found here at this [thread](https://osu.ppy.sh/community/forums/topics/565382), and previous discussions can be found [here](https://osu.ppy.sh/community/forums/topics/514122). 

Reviewer 2: was kinda already mentioned at the beginning, i think u can remove this paragraph

Author: kinda mentioned, but goes more indepth and link to the various other threads from before

> If any of this interests you or anyone you know, please go to our [forum announcement and rules thread](https://osu.ppy.sh/community/forums/topics/955356)! 

Reviewer 2: ```suggestion
If any of this interests you or anyone you know, please go to our [forum announcement and rules thread](https://osu.ppy.sh/community/forums/topics/955356) for more information!
```

> We're looking forwards to the future mappers of tomorrow!

Reviewer 2: ```suggestion
We're looking forward to the working with the mappers and modders of tomorrow!
```

Author: It should be fine as is, also holds the same tone as previous cycles before

Reviewer 2: at least `forwards` => `forward` then cuz its a phrasal verb instead of adverb. this wording is like saying "breaking downwards" instead of "breaking down"

Author: can do looking forward
